src/main/plugins/org.dita.html5/xsl/ui-d.xsl

```diff
@@ -26,7 +26,19 @@ See the accompanying LICENSE file for applicable license.
   <xsl:if test="ancestor::*[contains(@class,' ui-d/menucascade ')]">
    <xsl:variable name="uicontrolcount"><xsl:number count="*[contains(@class,' ui-d/uicontrol ')]"/></xsl:variable>
     <xsl:if test="$uicontrolcount&gt;'1'">
-      <xsl:text> > </xsl:text>
+      <xsl:variable name="a11y.text" as="text()?">
+        <xsl:call-template name="getVariable">
+          <xsl:with-param name="id" select="'a11y.and-then'"/>
+        </xsl:call-template>
+      </xsl:variable>
+      <abbr>
+        <xsl:if test="exists($a11y.text)">
+          <xsl:attribute name="title" select="$a11y.text"/>          
+        </xsl:if>
+        <xsl:call-template name="getVariable">
+          <xsl:with-param name="id" select="'#menucascade-separator'"/>
```

Comment:
To nitpick, `#menucascade-separator` is defined in PDF2 variable files. This means if you don't have PDF2 plugin installed, this will fail. Maybe it would be easier to move it from PDF2 to common variable files?